tests/test_Consumer.py

```diff
@@ -41,6 +41,17 @@ def dummy_assign_revoke(consumer, partitions):
     if msg is not None:
         assert msg.timestamp() == (TIMESTAMP_NOT_AVAILABLE, -1)
 
+    msglist = kc.consume(num_messages=10, timeout=0.001)
+    if len(msglist) == 0:
+        print('OK: consume() timeout')
+    else:
+        for msg in msglist:
+            if msg.error():
+                print('OK: consumer error: %s' % msg.error().str())
+                break
+
+        print('OK: consumed messages')
+
```

Comment:
Please add a failing test for invalid number of num_messages (e.g., -100 and > 1M)